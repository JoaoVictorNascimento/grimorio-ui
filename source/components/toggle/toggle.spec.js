import '../../../internals/test/helper';

import Toggle from './toggle-component';

/** @test {Toggle} */
describe('Toggle component', () => {
/** @test {Toggle#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(
        <Toggle />
      );
      expect(wrapper.length).toEqual(1);
    });
  });
});
