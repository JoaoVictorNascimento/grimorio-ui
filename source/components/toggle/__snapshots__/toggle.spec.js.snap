// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toggle component render correctly with props values 1`] = `
"<label htmlFor={123} className=\\"wrap isDisabled minhaClasse\\">
  <input value=\\"value\\" checked={true} onChange={[Function: mockConstructor]} type=\\"checkbox\\" id={123} className=\\"input\\" role=\\"switch\\" disabled={true} />
  <span className=\\"knob\\" />
  <span className=\\"track\\" />
</label>"
`;
